cloud-controller-manager/osc/oapi_sdk.go

```diff
@@ -17,144 +17,140 @@ limitations under the License.
 package osc
 
 import (
+	"context"
+	"errors"
 	"fmt"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	osc "github.com/outscale/osc-sdk-go/v2"
 )
 
 // ********************* CCM oscSdkCompute Def & functions *********************
 
-// oscSdkCompute is an implementation of the EC2 interface, backed by aws-sdk-go
+// oscSdkCompute is an implementation of the some EC2 interface and OSC Interface, backed by aws-sdk-go and osc-sdk-go
 type oscSdkCompute struct {
-	oapi *ec2.EC2
+	ec2    *ec2.EC2
```

Comment:
Why do you still need ec2 ?

Reply:
ec2 is also never used here in the file

Reply:
A commit is missing, the migration was done calls by calls. Now compute is migrated therefore yes it should be removed 👍